### PR TITLE
[IMP] crm: Clarify Kanban View

### PR DESCRIPTION
**Pull request by nboulif**
This commit will put more emphasis on both the revenue
and the contact attached to the opportunity.

Task ID 2240258

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/views/crm_lead_views.xml [hunk 1]
```diff
@@ -501,13 +501,12 @@
                                     <div>
                                         <strong class="o_kanban_record_title"><field name="name"/></strong>
                                     </div>
+                                    <div class="o_kanban_record_subtitle">
```
Comment (awa-odoo) [inline addons/crm/views/crm_lead_views.xml:504]: I would put the t-if on the div instead.
No reason not too.

Comment (nboulif) [reply addons/crm/views/crm_lead_views.xml:504]: So no need to display Customer if no expected revenue?

Comment (awa-odoo) [reply addons/crm/views/crm_lead_views.xml:504]: Oh you're right.
It is so badly formatted that I had not noticed.
Then maybe reformat to:
```xml
<div class="o_kanban_record_subtitle">
    <t t-if="record.planned_revenue.raw_value">
        <field name="planned_revenue" widget="monetary" options="{'currency_field': 'company_currency'}"/>
        <span t-if="record.partner_id.value"> - </span>
    </t>
    <span t-if="record.partner_id.value" t-esc="record.partner_id.value"></span>
</div>
```

For clarity?
Thanks for double checking!

Comment (nboulif) [reply addons/crm/views/crm_lead_views.xml:504]: Perfect. Thanks to you!

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**